features/polyExplorer/src/components/dataViz/purposesBarChart.jsx

```diff
@@ -145,18 +147,18 @@ const PurposesBarChart = ({ data, animation, onClick }) => {
                 .enter()
                 .append("foreignObject")
                 .attr("x", labelXPosition)
-                .attr("y", (d) => yScale(d.title) + gHeight)
+                .attr("y", (d) => yScale(d.title) + labelTitleYAdjust)
```

Comment:
is this value gHeight + 10 always? Only in this case? Totally unrelated?

Reply:
It is actually always `gHeight + 10`, I'll fix that.